src/react-components/src/__tests__/components/Pagination-test.tsx

```diff
@@ -0,0 +1,48 @@
+
+import { expect } from "chai";
+import { stub } from "sinon";
+import * as Enzyme from "enzyme";
+import * as React from "react";
+import * as Mocha from "mocha";
+
+import Pagination from "../../components/02-molecules/Menus/Pagination/Pagination";
+
+describe("Pagination Test", () => {
+
+  let wrapper: Enzyme.ShallowWrapper<{}, {}>;
+  let previousCallback;
+  let nextCallback;
+  let changeCallback;
+  let blurCallback;
+
+  beforeEach(() => {
+    previousCallback = stub();
+    nextCallback = stub();
+    changeCallback = stub();
+    blurCallback = stub();
+  });
+
+  it("Renders two buttons and a Select", () => {
+    wrapper = Enzyme.shallow(<Pagination
+      totalPages={10}
+      previousPageHandler={previousCallback}
+      nextPageHandler={nextCallback}
+      onSelectChange={changeCallback}
+      onSelectBlur={blurCallback} />
+    );
+    expect(wrapper.find("Button")).to.have.lengthOf(2);
+    expect(wrapper.find("Dropdown")).to.have.lengthOf(1);
+  });
+
+  it("Renders two buttons and a Select when there are zero pages", () => {
```

Comment:
This doesn't make much sense to me. Why render anything if the total pages is 0? You'll have a non-functional UI element.
![Screen Shot 2020-01-08 at 3 25 11 PM](https://user-images.githubusercontent.com/1280564/72013030-14131980-322b-11ea-9b96-8b8d0492a1eb.png)

Reply:
I'm still on a stubborn path where as much as possible I want problems to not break the page (hence not throwing an error).  This test serves as documentation for that case.  I think this is an approach we will want to revisit re: front end strategy - in general, when do we want to break the page and when do we want to let it slide?

Reply:
Hmmm it would be an approach where the DS _always_ renders the component and the app is responsible for appropriately rendering or not rendering it. I'm okay with that approach but it should be noted somewhere for reference.

Reply:
that's a good way of putting it - It would eventually go in some introductory documentation that goes through DS philosophy, but I'm gonna leave it here as a reminder to put a bullet point in the readme.